libs/s25main/ingameWindows/iwStatistics.cpp

```diff
@@ -278,29 +279,62 @@ void iwStatistics::Draw_()
     DrawStatistic(currentView);
 }
 
-void iwStatistics::DrawPlayerBoxes()
+void iwStatistics::DrawPlayerOverlays()
 {
-    unsigned short startX = 126 - numPlayingPlayers * 17;
-    DrawPoint drawPt = GetDrawPos() + DrawPoint(startX, 68);
+    unsigned short startX = 126 - (numPlayingPlayers - 1) * 17;
+    DrawPoint drawPt = GetDrawPos() + DrawPoint(startX - 17, 45 - 23);
     for(unsigned i = 0; i < gwv.GetWorld().GetNumPlayers(); ++i)
     {
         const GamePlayer& player = gwv.GetWorld().GetPlayer(i);
         if(!player.isUsed())
             continue;
 
-        if(activePlayers[i])
-        {
-            auto const playerBoxRect = Rect(drawPt, Extent(34, 12));
-            auto const playerStatusPosition =
-              DrawPoint(playerBoxRect.getOrigin() + playerBoxRect.getSize() / 2 + DrawPoint(0, 1));
-            DrawRectangle(playerBoxRect, player.color);
-            SmallFont->Draw(playerStatusPosition, getPlayerStatus(player), FontStyle::CENTER | FontStyle::VCENTER,
-                            COLOR_YELLOW);
-        }
+        // Draw the alliances at top of the player image
+        DrawPlayerAlliances(drawPt, i);
+        // Draw player boxes and player status at bottom
+        DrawPlayerBoxes(drawPt, player);
```

Comment:
Move the condition here is easier (and rename the function to singular):
```suggestion
        // Draw player color box and status at bottom
        if(activePlayers[i])
            DrawPlayerBox(drawPt, player);
```